python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -316,8 +316,10 @@ def _get_device(device: str) -> str:
        or type(device) == int
        or device is None):
         if device == Device.AUTO.value:
-            return (Device.CUDA.value if torch.cuda.is_available()
-                    else Device.CPU.value)
+            if torch.cuda.is_available():
+                return Device.CUDA.value
```

Comment:
I wonder if we should just set this to CPU (and leave a TODO comment) until we finish configuring CUDA?